lib/run.rb

```diff
@@ -16,3 +17,11 @@ def prompt_token_selection
 def get_player_token
     input = Input.new.get_input
 end
+
+def display_board
+    board = BoardPresenter.new.display_board
+end
+
+def assign_token
+    get_player_token == "X" ? "O": "X"
+end
```

Comment:
What's this function being used for right now? / Where is it being called?

Reply:
Not currently being called anywhere, but I need this assignment to start the game.